### DISCUSSION TITLE
The role of eccentricity in COMPAS

### DISCUSSION THREAD

**Discussion by hulin-Lee** [TeamCOMPAS/COMPAS#965] [Q&A] [answered]
According to my understanding, the radius at periapsis, i.e. a(1-e), of a stellar binary is crucial for binary evolution rather than the eccentricity. Thus, I expect the endings of binaries with same initial masses and  radius at periapsis would be the same. I run the following several cases
--initial-mass-1 106.30971651 --initial-mass-2 77.82751159 --metallicity 0.01 --eccentricity 0 --semi-major-axis 4 --random-seed 1683484032
--initial-mass-1 106.30971651 --initial-mass-2 77.82751159 --metallicity 0.01 --eccentricity 0.2 --semi-major-axis 5 --random-seed 1683484032
--initial-mass-1 106.30971651 --initial-mass-2 77.82751159 --metallicity 0.01 --eccentricity 0.5 --semi-major-axis 8 --random-seed 1683484032
--initial-mass-1 106.30971651 --initial-mass-2 77.82751159 --metallicity 0.01 --eccentricity 0.8 --semi-major-axis 20 --random-seed 1683484032
--initial-mass-1 106.30971651 --initial-mass-2 77.82751159 --metallicity 0.01 --eccentricity 0.9 --semi-major-axis 40 --random-seed 1683484032
 
The results of the first three runs (e<=0.5) are the same: Evolution ended by run duration: CHeB+MS at 3.479798Myr. The results of the last two runs are the same: Double compact object (BH+BH) at 4.436607Myr. I am wondering why evolution ended by run duration? Whether eccentricity plays others roles in COMPAS?

**Comment by reinhold-willcox** [comment, marked as answer]
Hi @hulin-Lee , 

Thanks for pointing this out, and apologies for the slow response. I agree with your assessment that this seems incorrect. We will take a look and get back to you shortly. 

Edit: I've just run the example binaries you posted, and they all appear to result in a BH+BH binary. I'm using v02.38.05. Can you verify for me if you are using this version?

**Comment by hulin-Lee** [comment]
Thanks! The version I used is v02.35.03. The phenomenon disappears after I updated to v02.38.05.

**Reply by reinhold-willcox** [reply]
I'm glad the problem has disappeared, whatever its cause. Thanks for your question!
